docker/irods/ubuntu/18.04/Dockerfile

```diff
@@ -1,39 +1,49 @@
 ARG DOCKER_TAG_PREFIX=wsinpg
-
 FROM $DOCKER_TAG_PREFIX/ub-18.04-base:latest
 
-ARG IRODS_VERSION=4.3.0-1~bionic
-ARG IRODS_PACKAGES_URL=https://packages.irods.org
-ARG UBUNTU_RELEASE=bionic
+ARG IRODS_VERSION="4.3.0"
 
 WORKDIR /opt/docker/irods
 
 COPY ./scripts/*.sh ./scripts/
 COPY ./config/* ./config/
 COPY ./patches/* ./patches/
 
-RUN apt-get update && \
+RUN echo "debconf debconf/frontend select Noninteractive" | debconf-set-selections && \
+    apt-get update && \
     apt-get install -q -y --no-install-recommends \
-    apt-transport-https \
+    apt-utils \
     ca-certificates \
+    curl \
     gpg \
     gpg-agent \
-    curl && \
-    curl -sSL $IRODS_PACKAGES_URL/irods-signing-key.asc | apt-key add - && \
-    echo "deb [arch=amd64] $IRODS_PACKAGES_URL/apt $UBUNTU_RELEASE main" | \
-    tee /etc/apt/sources.list.d/renci-irods.list && \
-    apt-get update && \
-    apt-get install -q -y --no-install-recommends \
-    postgresql \
+    lsb-release \
     jq \
+    netcat \
```

Comment:
It is only used to check if the service is running right? I think there is a less elegant way of doing something similar with echo `echo >/dev/tcp/localhost/1247` which may give you the same, not sure how it works with the redirection of echo. But if you want to try and then you don't need netcat.

Reply:
`netcat`'s working and isn't a burden, so I'll leave this as is, rather than  experiment.